templates/bottlerocket_user_data.tpl

```diff
@@ -1,7 +1,8 @@
 %{ if enable_bootstrap_user_data ~}
+${pre_bootstrap_user_data ~}
 [settings.kubernetes]
-"cluster-name" = "${cluster_name}"
 "api-server" = "${cluster_endpoint}"
 "cluster-certificate" = "${cluster_auth_base64}"
+"cluster-name" = "${cluster_name}"
+${post_bootstrap_user_data ~}
```

Comment:
I'm curious why these two changes were necessary?

Reply:
We don't really use Bottlerocket, it was mostly for consistency with the Linux template we use.
It's not necessary and can be removed to minimize differences compared to the upstream.

Reply:
Removed in https://github.com/pinterest/terraform-aws-eks/pull/3/commits/e3413695d157974289581b7434220055c456f8f5